tools/src/sign_image.rs

```diff
@@ -1,4 +1,4 @@
-use std::io::{Error, ErrorKind, Read, Write};
+use std::io::{Read, Write};
```

Comment:
strange, when this patch was merged I get this error duing build:

```
   Compiling tools v0.1.0 (F:\largework\rust-win\code\xous-core\tools)
error[E0433]: failed to resolve: use of undeclared type `ErrorKind`
  --> tools\src\sign_image.rs:38:37
   |
38 |             .map_err(|_| Error::new(ErrorKind::Other, "error parsing current Git rev"))?
   |                                     ^^^^^^^^^ use of undeclared type `ErrorKind`

error[E0433]: failed to resolve: use of undeclared type `Error`
  --> tools\src\sign_image.rs:38:26
   |
38 |             .map_err(|_| Error::new(ErrorKind::Other, "error parsing current Git rev"))?
   |                          ^^^^^ use of undeclared type `Error`
   |
help: consider importing one of these items
   |
1  | use clap::Error;
   |
1  | use core::error::Error;
   |
1  | use core::fmt::Error;
   |
1  | use csv::Error;
   |
     and 3 other candidates
```

Not sure what's going on here but I'm reverting the patch...